packages/phone-number-privacy/monitor/src/resources.ts

```diff
@@ -0,0 +1,49 @@
+import { EncryptionKeySigner } from '@celo/identity/lib/odis/query'
+import { AuthenticationMethod } from '@celo/phone-number-privacy-common'
+import {
+  ensureLeading0x,
+  normalizeAddressWith0x,
+  privateKeyToAddress,
+} from '@celo/utils/lib/address'
+import 'isomorphic-fetch'
```

Comment:
I'm not sure why this line was in the combiner, but not sure if we need it here

Reply:
removed